src/main/kotlin/no/nav/familie/ks/sak/config/BisysConfig.kt

```diff
@@ -0,0 +1,51 @@
+package no.nav.familie.ks.sak.config
+
+import no.nav.familie.ks.sak.sikkerhet.SikkerhetContext
+import no.nav.familie.sikkerhet.OIDCUtil
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Profile
+import org.springframework.web.filter.OncePerRequestFilter
+import javax.servlet.FilterChain
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+
+@Configuration
+@Profile("!integrasjonstest")
+class BisysConfig(
+    private val oidcUtil: OIDCUtil,
+    private val rolleConfig: RolleConfig
+) {
+
+    @Bean
+    fun bisysFilter() = object : OncePerRequestFilter() {
+        override fun doFilterInternal(request: HttpServletRequest, response: HttpServletResponse, filterChain: FilterChain) {
+            val clientNavn = oidcUtil.getClaim("azp_name")
```

Comment:
Her vil det nok kastes exception hvis den ikke finner azp_name. Kanskje håndtere med en feilmelding om at token må være configurert med 
claims:
        extra:
          - "azp_name"

I så fall så tenker jeg dette vil fungere. Så lenge man er sikker på at appen heter noe med bidrag

Reply:
Ja, feilmeldingen kan kastes. Hvis app-en heter noe annet enn bidrag, kommer ikke forespørselen til filter og feiler det på Azure AD nivå fordi app-en er konfigurert som applikasjon i inbound policy.

Reply:
så litt nærmere nå. getClaim() metode kaster feilmeldingen automatisk this.jwtError("Fant ikke claim '" + claim + "' i tokenet");